.github/workflows/native-build.yml

```diff
@@ -44,6 +47,13 @@ jobs:
             compiler-version: 11
             additional-cmake-options: "-DUSE_CPP_17_BACKPORTS=ON"
             build-type: Release
+          - compiler: clang
+            compiler-version: 15
+            additional-cmake-options: "-DUSE_CPP_17_BACKPORTS=ON -DCMAKE_CXX_STANDARD=17 -DCMAKE_CXX_FLAGS='-ferror-limit=0'"
+            build-type: Debug
+            expensive-tests: false
+            continue-on-error: true
+            use-keep-going: true
```

Comment:
I have just started the pipeline, this would be an initial step to track the progress.